src/components/Home/QR/index.js

```diff
@@ -1,25 +1,47 @@
 import React, { Component } from 'react';
 import {
-  StyleSheet, Text, View,
+  Text, View,
 } from 'react-native';
-
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
-  },
-});
+import { connect } from 'react-redux';
+import { RNCamera } from 'react-native-camera';
+import { checkQRDispatch } from 'datalayer/actions/qr.action';
+import NavigationWithoutProps from 'utils/navigationWithoutProps';
+import styles from './index.styles';
 
 class QR extends Component {
```

Comment:
Check if camera will be unmounted after we leave this screen

Reply:
Cu yen tam. Trust me